common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/auth/AuthenticationClient.java

```diff
@@ -31,4 +31,9 @@ public interface AuthenticationClient
 {
 
     AuthenticationResult authenticate(String tokenUri, ClientRegistration clientRegistration);
+
+    default AuthenticationResult authenticate(String clientRegistrationId)
```

Comment:
Generally, this method is planned to be the only one used across the applications and the one above will gradually be replaced by this one.